docs/devices/W599001.md

```diff
@@ -0,0 +1,5 @@
+<!-- Notes BEGIN -->
+## Notes
+Cluster 64516 is not implemented.
```

Comment:
It is not clear, what this means for the user, can you explain it?

Reply:
Well, it is not clear for me either... but I suppose I could make that more explicit...